src/tasks/WasmAppBuilder/WasmAppBuilder.csproj

```diff
@@ -26,11 +26,11 @@
     <!-- excludeassets prevents the older version of MLC from MSBuild from trampling the newer version we need -->
     <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildVersion)" ExcludeAssets="runtime" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildTasksCoreVersion)" ExcludeAssets="runtime" />
-    <!-- we need to explicitly reference MLC 8.0 or newer for function pointer support,
+    <!-- we need to explicitly reference System.Reflection.MetadataLoadContext 8.0 or newer for function pointer support,
          because the default version pulled in from MSBuild is 6.0. -->
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataToolsetVersion)" />
-    <PackageReference Include="System.Reflection.MetadataLoadContext" Version="$(SystemReflectionMetadataLoadContextVersion)" />
-    <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
+    <PackageReference Include="System.Reflection.MetadataLoadContext" Version="$(SystemReflectionMetadataLoadContextToolsetVersion)" />
```

Comment:
MetadataDataLoadContext isn't a shared toolset dependency. It should just use a normal version property.

Reply:
We can't since then darc will update the version to a 9.0.0-preview one

Reply:
Add the `Pinned=true` attribute to the entry in Version.Details.xml. cc @mmitche

Reply:
How did this not get upgraded with darc before when it was the 8.0.0 version?  I think pinning will cause SB to use an SBRP because it won't lift it anymore.

Reply:
That's a new dependency, just recently introduced.